gatsby/content/projects/2020-12-04-neochat.mdx

```diff
@@ -0,0 +1,59 @@
+---
+layout: projectimage
+title: NeoChat
+categories: 
+ - client
+thumbnail: /docs/projects/images/neochat_thumb.png
+description: A Matrix client for desktop and mobile
+author: Carl Schwan, Tobias Fellas
+maturity: Beta
+language: C++/Qt QML
```

Comment:
```suggestion
language: C++/QML
```

I think it would be better to list this in the same category as Spectral and Quaternion rather than a new category